apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/style.scss

```diff
@@ -0,0 +1,8 @@
+.site-description {
```

Comment:
Should we namespace this class in order to avoid potential conflicts with other styles?